src/sentry/models/integration.py

```diff
@@ -13,6 +13,21 @@
 from sentry.signals import integration_added
 
 
+class PagerDutyServiceProject(Model):
+    __core__ = False
+
+    project = FlexibleForeignKey("sentry.Project", db_index=False, db_constraint=False)
```

Comment:
@MeredithAnya why do we have `db_index=False` here?

Reply:
since we have the unique index on `(project_id, organization_integration_id)` we don't really need another one on the project